### ISSUE TITLE
Unable to copy responses to clipboard

### ISSUE THREAD

**Issue by ppearson111** [Huachao/vscode-restclient#4]
Cool plugin for vscode, will be very handy, thanks!  One issue I've seen is this:

Not sure if this is an issue with the extension or with VSCode itself... but I am unable to copy responses to the clipboard.  This is very useful for copying redirect URL's in 30x http responses from the rest service that is being tested.  

Also, would it be possible to have an option to automatically handle 3xx redirect message responses from the server?

**Comment by Huachao** [comment]
@ppearson111 Thanks for your questions.
1. Unable to copy the response is the limitation of VSCode, I have created the issue [here](https://github.com/Microsoft/vscode/issues/6924). I think it will be solved soon since they marked the milestone `June 2016`
2. We have already handled 3xx redirect response with an options `rest-client.followredirect`, by default it is true, that means it will automatically redirect according to the location header of 3xx response.
